### PR TITLE
Update pinning markup

### PR DESCRIPTION
**Pull request by sourcier**
🐿 v2.7.0

### CODE REVIEW

File: styles/_myft-card.scss [hunk 1]
```diff
@@ -70,3 +73,10 @@
 		color: getColor('black-80');
 	}
 }
+
+.topic-card__myft-footer__divider {
```
Comment (GlynnPhillips) [inline styles/_myft-card.scss]: This looks a bit weird to me, looks like you have two "elements".

File: templates/concept.html [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {{#concept @this}}
-<section data-trackable="concept" aria-label="{{name}}" {{#if responsiveGrids}}data-o-grid-colspan="{{responsiveGrids}}"{{/if}} class="topic-card__myft {{#unless items.length}}topic-card--empty{{/unless}}{{#if lazyLoadImages}} topic-card--lazy-loading-images{{/if}} {{classes}}">
+<section data-trackable="concept" aria-label="{{name}}" {{#if responsiveGrids}}data-o-grid-colspan="{{responsiveGrids}}"{{/if}} class="topic-card__myft {{#unless items.length}}topic-card--empty{{/unless}}{{#if lazyLoadImages}} topic-card--lazy-loading-images{{/if}} {{classes}}" data-prioritised="{{#if prioritised}}true{{else}}false{{/if}}">
```
Comment (GlynnPhillips) [inline templates/concept.html]: As data attributes are boolean attributes could this just be reduced to this `{{#if prioritised}}data-prioritised{{/if}}`

File: templates/concept.html [hunk 3]
```diff
@@ -74,7 +75,7 @@
 						<input type="hidden" value="http://www.ft.com/ontology/concept/Concept" name="directType">
 					{{/if}}
 					<button
-						class="o-buttons {{#if prioritised}}o-buttons--primary{{/if}} o-buttons-icon o-buttons-icon--icon-only myft-ui__button--pin"
+						class="myft-ui__pin-button"
```
Comment (GlynnPhillips) [inline templates/concept.html]: Not sure if we should use the namespace as the block level. So maybe something more like `myft-pin-button`

File: styles/_myft-card.scss [hunk 4]
```diff
@@ -70,3 +73,10 @@
...
+	height: 18px;
+	width: 1px;
+	background: getColor('black-30');
```
Comment (benbarnett) [inline styles/_myft-card.scss]: I wonder if this should have the same border colour as the other dividers `black-10`?

**Comment by sourcier** [comment]
@benbarnett @GlynnPhillips made recommended changes.

**Pull request merged**